.github/workflows/ci-conda.yml

```diff
@@ -95,6 +94,12 @@ jobs:
         shell: bash -l {0}
         run: ./sage -t --all -p0
 
+      # We keep this step for now to make sure that the configure-based setup still works.
+      - name: Test configure
+        shell: bash -l {0}
+        run: |
+          ./configure --enable-build-as-root --with-python=$CONDA_PREFIX/bin/python --prefix=$CONDA_PREFIX --enable-system-site-packages $(for pkg in $(./sage -package list :standard: --has-file spkg-configure.m4 --has-file distros/conda.txt --exclude rpy2); do echo --with-system-$pkg=force; done) || true
```

Comment:
What is this, why re-introduce this long command line that is invoked by sage-conf_conda?

Reply:
The conda-ci never used the sage conf package (and its not the point of this PR to change that).